README.md

```diff
@@ -140,6 +140,44 @@ above; although it is conceivable that such a composite project may define its
 own services, it is unlikely and ill-advised because no one likes blurred lines
 in architectural diagrams. Just look at the Leaning Tower of Pisa.
 
+#### Additional Uberjars
+
+Some projects might have a use case for automatically fetching an versioned
+jar from an external repository that needs to be placed into the package and be
+available after installation.
+
+The `:additional-uberjars` setting can be set to a list of project coordinates
+that will be resolved and have uberjars built from them. For example:
+
+```clojure
+{:lein-ezbake {
+  :additional-uberjars [[puppetlabs/puppetserver "2.7.2"]]
+...}}
+```
+
+This would result in a puppetserver uberjar being built, and the jar being placed in the
+same directory as your project's own uberjar after installation.
+
+The filename of the built uberjar is determined by what value `:uberjar-name`
+is set to in its `project.clj`
+
+ezbake will attempt to resolve the coordinates of these external uberjars using
+the repositories specified in the `:repositories` key of your project.clj
+
+#### cli-defaults.sh
+`cli-defaults.sh` is intended to be a file from which default shell script
+variables can be defined by a project, and be loaded by ezbake's `cli-app`
+script. That is, the script which is run when commands like
+`service my-project start` are called.
+
+This differs from the `default` files, which also get loaded by `cli-app`, in
+that a project has no control over what goes into `default`. `cli-defaults.sh`
+is under the control of your project.
+
+For `cli-defaults.sh` to be used, it simply needs to exist at
```

Comment:
I haven't tried that out but my guess is that the behavior would be similar to what happens with other resources with the same location in an dependency jar as in the downstream composite jar.  In those cases, I've observed that the downstream, composite one "wins" (is the one that gets bundled in the package).  If that is the behavior, does that seem reasonable to you at least for now?  I'd like to avoid having to complicate the logic by aggregating multiple possible files from each of the component jars and sourcing them all in some order from the startup scripts, etc.  Would be good to at least know what would happen in this situation, though - e.g., that the presence of multiple files in the different jars wouldn't cause an ezbake build to fail.